node/service/chain-specs/example.json

```diff
@@ -95,6 +95,36 @@
           ]
         ]
       },
+      "democracy": {
```

Comment:
The `democracy` pallet was removed when we introduced `OpenGov`, you should remove it from the `example.json` file.